packages/block-library/src/breadcrumbs/index.php

```diff
@@ -179,6 +179,21 @@ function block_core_breadcrumbs_get_archive_breadcrumbs() {
 		$month = get_query_var( 'monthnum' );
 		$day   = get_query_var( 'day' );
 
+		// Fallback to 'm' query var for plain permalinks.
+		// Plain permalinks use ?m=YYYYMMDD format instead of separate query vars.
+		if ( ! $year ) {
+			$m = get_query_var( 'm' );
+			if ( $m ) {
+				$year = substr( $m, 0, 4 );
+				if ( strlen( $m ) >= 6 ) {
```

Comment:
Nitpick: I don't think we need to check the string length, as if the string is smaller and we do $month = substr( $m, 4, 2 ); we will just get an empty string, on src/wp-includes/general-template.php they just do:
```
	if ( is_archive() && ! empty( $m ) ) {
		$my_year  = substr( $m, 0, 4 );
		$my_month = substr( $m, 4, 2 );
		$my_day   = (int) substr( $m, 6, 2 );
}
```
Maybe we can use the same code and link to it in a comment?

This is offcourse just a suggestion, and not a blocker, as what we have is perfectly fine.

Reply:
>if the string is smaller and we do $month = substr( $m, 4, 2 ); we will just get an empty string

Yeah, the checks were not needed here. I updated.